README.md

```diff
@@ -0,0 +1,112 @@
+Inheritance Shares Calculator
+
+This application calculates the inheritance shares for sons, daughters, and other family members based on the total inheritance amount and family structure.
+Features
+
+    Inheritance calculation for sons, daughters, wife, mother, and father.
+    Error handling for invalid input types.
+    Unit testing is included to ensure correctness of the logic.
+
+Requirements
+
+    Python 3.x
+    unittest for unit testing.
+
+How to Run the Application
```

Comment:
Please give the instructions how to setup a virtual environment. Also list down the required dependencies in `requirements.txt` and give instructions how to install them via `requirements.txt.